packages/app-project/src/screens/ProjectHomePage/components/Hero/components/Background/Background.js

```diff
@@ -1,39 +1,45 @@
-import { withResponsiveContext } from '@zooniverse/react-components'
-import { string } from 'prop-types'
+import { number, shape, string } from 'prop-types'
 import React from 'react'
-import styled from 'styled-components'
+import styled, { withTheme } from 'styled-components'
 
 const Img = styled.img`
-  height: 100%;
-  object-fit: cover;
-  object-position: 0 50%;
-  width: 100%;
-
-  ${props => props.screenSize === 'small' && `
-    height: auto;
+    height: 230px;
```

Comment:
Has this crept in from another PR?

Reply:
ugh, looks like it

Reply:
Version changes from the grommet update have appeared in this PR too.

Reply:
Okay, I've fixed the history